reference_designs/ipu-xrt/matrix_multiplication_column/aie2.py

```diff
@@ -0,0 +1,261 @@
+#
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# (c) Copyright 2023 AMD Inc.
+
+import aie
+from aie.ir import *
+from aie.dialects.func import *
+from aie.dialects.scf import *
+from aie.dialects.aie import *
+from aie.dialects.aiex import *
+from aie.dialects.extras import memref, arith
+from aie.util import mlir_mod_ctx
```

Comment:
```suggestion
from aie.extras.dialects.ext import memref, arith
from aie.extras.context import mlir_mod_ctx
```

Reply:
@fifield is it safe to add `reference-designs` to the ryzenai CI flow?

Reply:
> @fifield is it safe to add `reference-designs/` to the ryzenai CI flow?

yes. I thought it already was, but maybe not.